EIPS/eip-5503.md

```diff
@@ -0,0 +1,176 @@
+---
+eip: 5503
+title: Refundable Token Standard
+author: <admin@startfund.io>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-16
+---
+
+## Abstract
+The value of a token can be the total sum of the linked currency’s value. For example, in the Token issuing process, the issuer can receive money from buyers( or investors) and transfer issuing token to buyers. . If the offering process is completed, there is no issue. But buyers can change their plan, or the offering is failed(or be canceled) cause of misfitting the compliance rules or other rules. There is no way guarantee to pay back (refund) to the buyer in the on-chain network.
```

Comment:
[editorial] typos, double periods ... to publish the draft, please conduct a minimum effort of spell and grammar check.

Reply:
[editorial] content-wise, it makes me feel a large part of the content here belongs to motivation part rather than abstract. Can you write the abstract to give an overview of what is being specified and a very simple purpose statement?

Reply:
Made `Abstract` part more clear for overview.